src/audio/Audio.d.ts

```diff
@@ -17,6 +17,7 @@ export class Audio extends Object3D {
 	loop: boolean;
 	startTime: number;
 	offset: number;
+	duration: number;
```

Comment:
If the default value is `undefined`, we should do this:
```
duration: number | undefined;
```

Reply:
Fixed, thanks for the catch!